qa/L0_perf_resnet/run_test.sh

```diff
@@ -69,16 +70,21 @@ if [ "$ARCH" == "aarch64" ]; then
     fi
 else
     PERF_CLIENT=../clients/perf_client
+    if [ "$MODEL_FRAMEWORK" == "savedmodel_trt" ] || [ "$MODEL_FRAMEWORK" == "onnx_trt" ]; then
+        MEASUREMENT_WINDOW=20000
```

Comment:
Instead of increasing measurement windows for such model, how about adding a mode in perf analyzer that sends the request to the model and wait for response before starting the actual profiling. 
This will reduce the overall runtime.  There will be no need to run 3-4 cycles of 20 sec (runtime: 60s) as the same can take 20sec of init time and 15sec(3 cycles of default 5sec) of profiling (runtime: 45sec)

Reply:
@tanmayv25 can you also try to use 20000 as the measurement window for onnx and onnx-trt?
Additionally can you check if this slowdown impacted aarch64 as well?

Reply:
I think the PR is already trying to use 20000 sec measurement window for onnx-trt,
Do you see failures on aarch64 on CI?

Reply:
You are right. I missed that. I do however see aarch64 failures for perf resnet on master because of onnx-trt even with 20000 ms as measurement window. This occurs even with concurrency 1.